static/src/javascripts/components/raven-js/raven.js

```diff
@@ -1835,4 +1835,4 @@ if (typeof define === 'function' && define.amd) {
     define('raven', [], function() { return Raven; });
 }
 
-})(this);
+})(window);
```

Comment:
Yeah, it's a weird pattern!

The Windows phone error was:

>[Can't execute code from a freed script](https://sentry.io/the-guardian/client-side-prod/issues/181594029/)

We were not able to debug this issue successfully, but I'll take a look at Raven and see if a similar method call occurs.